spec/features/course/assessment/question/multiple_response_management_spec.rb

```diff
@@ -139,10 +139,10 @@
 
         visit edit_path
         options.each do |option|
-          click_button 'Add a new response'
+          click_button 'Add a new choice'
```

Comment:
Why did you change this?

Reply:
sorry I looked into the CircleCI and they indicated this testcase was not passed. So I tried to change it, but apparently also still not pass. Now I have reverted it back

Reply:
You can always try it locally after you see if theres any failed test cases in circleci